wg-api/README.md

```diff
@@ -29,8 +29,7 @@ Oversees public API design based on project principles.
 ## Meeting Schedule
 
 * Biweekly Rotating Times:
-  * 5 PM PT monthly [calendar](https://calendar.google.com/event?action=TEMPLATE&tmeid=MGdmdmxudG9vOXFianFmcHM3NDBiN2trMzhfMjAxOTEwMDhUMDAwMDAwWiBjYzM3dmx1c2w2Z3Y2dnVvbjhrZGNqYzc3a0Bn&tmsrc=cc37vlusl6gv6vuon8kdcjc77k%40group.calendar.google.com&scp=ALL)
-  * 9 AM PT monthly + offset [calendar](https://calendar.google.com/event?action=TEMPLATE&tmeid=MWdqdmZiZ2RoY3FxNDg5M3BlN2c1cDg0bnBfMjAxOTA5MjZUMTYzMDAwWiBjYzM3dmx1c2w2Z3Y2dnVvbjhrZGNqYzc3a0Bn&tmsrc=cc37vlusl6gv6vuon8kdcjc77k%40group.calendar.google.com&scp=ALL)
+  * 4 PM PT bi-weekly [calendar](https://calendar.google.com/event?action=TEMPLATE&tmeid=MGdmdmxudG9vOXFianFmcHM3NDBiN2trMzhfMjAxOTEwMDhUMDAwMDAwWiBjYzM3dmx1c2w2Z3Y2dnVvbjhrZGNqYzc3a0Bn&tmsrc=cc37vlusl6gv6vuon8kdcjc77k%40group.calendar.google.com&scp=ALL)
```

Comment:
This calendar link needs to be updated, but I'm not sure how to generate a public link for the event